libcxx/test/support/test_iterators.h

```diff
@@ -1180,8 +1198,8 @@ struct Proxy {
   // Calling swap(Proxy<T>{}, Proxy<T>{}) would fail (pass prvalues)
 
   // Compare operators are defined for the convenience of the tests
-  friend constexpr bool operator==(const Proxy&, const Proxy&)
-    requires (std::equality_comparable<T> && !std::is_reference_v<T>)
+  friend constexpr bool operator==(const Proxy& lhs, const Proxy& rhs)
```

Comment:
Is there a reason you added names to these parameters?

Reply:
I'll just remove them